doc/validation.md

```diff
@@ -2,11 +2,20 @@
 
 BAPCtools does 3 types of validation, one more than the spec (https://www.kattis.com/problem-package-format/spec/problem_package_format#input-validators):
 
+1. input validation, which validates the `.in` file for every test case, typically for syntax, format, and ranges
```

Comment:
add correctness here? Or some other word that contains e.g. checking that a graph is connected.